packages/kbn-test/src/functional_tests/cli/run_tests/__snapshots__/args.test.js.snap

```diff
@@ -0,0 +1,137 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`display help for run tests CLI displays as expected 1`] = `
+"Run Functional Tests
+
+Usage:
+  node scripts/functional_tests --help
+  node scripts/functional_tests [--config <file1> [--config <file2> ...]]
+  node scripts/functional_tests [options] [-- --<other args>]
+
+Options:
+  --help                         Display this menu and exit. 
+  --config <file>                Pass in a config. Can pass in multiple configs. 
+  --esFrom <snapshot|source>     Build Elasticsearch from source or run from snapshot. Default: snapshot
+  --kibana-install-dir <dir>     Run Kibana from existing install directory instead of from source. 
```

Comment:
__nit:__ it's the only parameter that uses dashes now, maybe we can make it `--kibanaInsallDir` just for consistency?